pkg/extensions/enterprise_imports.go

```diff
@@ -18,6 +19,7 @@ import (
 	_ "github.com/go-jose/go-jose/v3"
 	_ "github.com/gobwas/glob"
 	_ "github.com/googleapis/gax-go/v2"
+	_ "github.com/grafana/grafana/pkg/build"
```

Comment:
This shouldn't be added, I know it shows up as a diff when you `make update-workspace` with Enterprise linked, but it is a known problem, you have to remove that line manually from go.mod/go.sum 😞